### ISSUE TITLE
[SUGGESTION] Add .png to upload

### ISSUE THREAD

**Issue by Chuuuckie** [Steemhunt/web#302]
I am having problems uploading .png files to steemhunt draft interface. The GIF files are working properly and so is JPEG.

**Comment by Chuuuckie** [comment]
![unnamed](https://user-images.githubusercontent.com/36227271/49784152-48ca4e00-fd57-11e8-9740-a3186913bc19.png)


![unnamed 1](https://user-images.githubusercontent.com/36227271/49784158-4cf66b80-fd57-11e8-9d34-37fa0a9adc54.png)


![unnamed 2](https://user-images.githubusercontent.com/36227271/49784163-4f58c580-fd57-11e8-9fbb-f5df79485b1e.png)

Tried this images - I dragged it into the upload section, it wouldnt upload. This error has been already for months, and I thought SteemHunt is not accepting .png files. Im using chrome.
